### PR TITLE
SFD-3627 Create Performance Tests

### PR DESCRIPTION
**Pull request by gwaardenburg**

### CODE REVIEW

File: project.code-workspace [hunk 1]
```diff
@@ -193,10 +193,7 @@
       "titleBar.activeForeground": "#e7e7e7",
       "titleBar.inactiveBackground": "#eb542499",
       "titleBar.inactiveForeground": "#e7e7e799",
```
Comment (daonhs) [inline project.code-workspace:195]: I think we should remove this from the commit

**Comment by daonhs** [comment]
is 'test/jmeter/properties/placeholder.user.properties' needed ?

**Review by daonhs — CHANGES_REQUESTED** [review]
Looks good to me, just remember to squash the commits

File: project.code-workspace (hunk 1 above)
Comment (gwaardenburg) [reply project.code-workspace:195]: I think this came in as part of the rebase I did.

**Review by ArifNhs — APPROVED** [review]
Approved

**Comment by gwaardenburg** [comment]
> 

This is part of the template and the build scripts looks for it. We can just leave it how it is for now. We might want to add properties at a later date

**Review by daonhs — APPROVED** [review]

**Review by jope1 — APPROVED** [review]

**Pull request merged**